### ISSUE TITLE
Sensor Labels

### ISSUE THREAD

**Issue by tompawompa** [Blackymas/NSPanel_HA_Blueprint#1437]
I have a number of zigbee temp/humidity sensor that I set up all at once, and they're names are th_1, th_2, etc...

It would be nice to be able to edit their labels in the climate page.

**Comment by edwardtfn** [comment]
Do you mean having a different sensor per climate page?

**Comment by tompawompa** [comment]
No, where you can configure additional sesnsors to display on the climate page. You can edit the label color, but not the label itself. There is no text displaying on the page at all next to the sensors actually. 


![Screenshot_20231219_062304_Home Assistant](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/95145927/fddf166d-7d63-4bc2-afc3-2227ca023dd6)
![20231219_062954](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/95145927/dd2ed3d3-c0a7-480e-816a-f7dc1c1804d5)

**Comment by edwardtfn** [comment]
But no label is shown for those custom sensors.
Do you mean the page title?

**Comment by tompawompa** [comment]
Sorry, I mean it would be nice to be able to label them. Even in lieu of an icon, if space is an issue.

**Comment by edwardtfn** [comment]
Ok, I will look at this later.
